data/common.css

```diff
@@ -13,7 +13,7 @@ main>.container {
 }   
 
 .footer {
-    background-color: #f5f5f5f5;
+    background-color: #f5f5f58c;
```

Comment:
Something looks _off_ about this color code (or you have a really funky monitor!)